CesiumGeometry/include/CesiumGeometry/Ray.h

```diff
@@ -2,6 +2,7 @@
 
 #include "Library.h"
 
+#include <glm/mat4x4.hpp>
```

Comment:
Better to include `<glm/fwd.hpp>`. It might be possible to remove the `glm/vec3.hpp` below as well.

Reply:
> Better to include <glm/fwd.hpp>

Why?

Reply:
Header files should usually always use forward declarations when available (mainly due to shorter compilation times and less risk of namespace issues, circular dependencies and such)

Reply:
Makes sense. I've changed this, as well as a handful of other header file occurrences that could do the same thing.